server/chat-plugins/mafia.ts

```diff
@@ -4026,7 +4144,8 @@ export const commands: Chat.ChatCommands = {
 			`/mafia spirit [player] - Kills a player, but allows them to vote still. Requires host % @ # &`,
 			`/mafia spiritstump [player] - Kills a player, but allows them to talk and vote during the day. Requires host % @ # &`,
 			`/mafia kick [player] - Kicks a player from the game without revealing their role. Requires host % @ # &`,
-			`/mafia revive [player] - Revive a player who died or add a new player to the game. Requires host % @ # &`,
+			`/mafia revive [player] - Revive a player who was eliminated. Requires host % @ # &`,
```

Comment:
To make it consistent with other messages "Revives"

Reply:
What exactly do you want changed? This isn't a code change suggestion just a message.

Reply:
Minor thing, but thinking that revives is better than revive here, for consistency. If you decide not to, I'd not mind though.

Reply:
Fair enough.